executor/modes/local/runner.py

```
@@ -0,0 +1,515 @@
+
+            # Store reference for cleanup
+            self._file_handler = file_handler
+
+            # Log startup info (this will go to both console and file)
+            logger.info(f"File logging enabled: {log_file}")
+            logger.info(
+                f"Log rotation: max {config.WEGENT_EXECUTOR_LOG_MAX_SIZE}MB, "
+                f"keep {backup_count} backups"
+            )
+
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)